app/src/pages/CollectionPage/CollectionPage.tsx

```diff
@@ -23,6 +24,12 @@ export const CollectionPage = (): React.ReactElement => {
     updateCollection();
   }, [updateCollection]);
 
+  const COLLECTION_TOKENS = [
+    new CollectionToken('0x1A257a5b37AC944DeF62b28cC5ec6c437178178c', '38123', 'Robo Ooga #38123', 'https://mekaapes.s3.amazonaws.com/images/38123.png', 'sold 12/12/21 for 0.3'),
```

Comment:
these descriptions are not valid - "sold 12/12/21 for 0.3" is NOT a valid description. this should show you the text under the title needs to be passed in cos its not part of the token...